src/charm.py

```diff
@@ -153,6 +156,25 @@ def __init__(self, *args):
     ######################
     # UTILITY PROPERTIES #
     ######################
+    @property
+    def _grafana_source_extra_fields(self) -> Dict[str, Any]:
```

Comment:
I'd get rid of this method and directly pass self._build_service_graph_config() to grafana_source.
we can move the comment there.

Reply:
I was thinking of keeping it like that, as we'd add other stuff as well other than svc graphs (e.g: logs correlation)